slackdump.go

```diff
@@ -1,233 +1,277 @@
-package slackdump

```

Comment:
I don't know what happened here looks like it was brutally reformatted, so that it's own mother does not recognise it.